### PR TITLE
New link

### PR DESCRIPTION
**Pull request by adfost**
Fixed comments from old PVC tenants link PR, adding link to tenant on PVC page 
Screen Shot 2021-07-22 at 3 22 04 PM

### CODE REVIEW

File: portal-ui/src/screens/Console/Storage/StoragePVCs.tsx [hunk 1]
```diff
@@ -127,7 +138,10 @@ const StorageVolumes = ({
             },
             {
               label: "Volume",
```
Comment (dvaldivia) [inline portal-ui/src/screens/Console/Storage/StoragePVCs.tsx:140]: Change the label to `Tenant`

File: restapi/embedded_spec.go [hunk 2]
```diff
@@ -2886,62 +2886,6 @@ func init() {
         }
       }
     },
```
Comment (dvaldivia) [inline restapi/embedded_spec.go:2888]: these should not be deleted, please rebase and do `make swagger-gen`

**Review by dvaldivia — CHANGES_REQUESTED** [review]

**Review by dvaldivia — DISMISSED** [review]

File: portal-ui/src/screens/Console/Storage/StoragePVCs.tsx [hunk 3]
```diff
@@ -83,7 +84,17 @@ const StorageVolumes = ({
...
+  const tableActions = [
+      {
+          type: "view",
+          onClick: (record: any) => {
+              history.push(`/namespaces/${record.namespace}/tenants/${record.tenant}`);
+              },
+      },
+  ];
+
+  // @ts-ignore
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Storage/StoragePVCs.tsx:96]: Please, do not use this, this is a bad practice

File: portal-ui/src/screens/Console/Storage/StoragePVCs.tsx [hunk 4]
```diff
@@ -142,7 +156,7 @@ const StorageVolumes = ({
           isLoading={loading}
           records={filteredRecords}
           entityName="PVCs"
-          idField="name"
+          idField="id"
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Storage/StoragePVCs.tsx:159]: `id` field doesn't exist under `IStoragePVCs` type. Please change this for `name` or other variable

**Review by bexsoft — CHANGES_REQUESTED** [review]

File: portal-ui/src/screens/Console/Storage/StoragePVCs.tsx [hunk 5]
```diff
@@ -83,7 +84,16 @@ const StorageVolumes = ({
...
 
-  return (
+  const tableActions = [
+      {
+          type: "view",
+          onClick: (record: any) => {
+              history.push(`/namespaces/${record.namespace}/tenants/${record.tenant}`);
+              },
+      },
+  ];
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Storage/StoragePVCs.tsx:94]: Please apply prettier to these lines. (You can generate assets)

File: portal-ui/src/screens/Console/Storage/StoragePVCs.tsx [hunk 6]
```diff
@@ -126,8 +136,11 @@ const StorageVolumes = ({
               width: 120,
             },
             {
-              label: "Volume",
-              elementKey: "volume",
+              label: "Tenant",
+              renderFullObject: true,
+              renderFunction: (record: any) => {
+                return `${record.namespace}/${record.tenant}`
+              },
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Storage/StoragePVCs.tsx:143]: Function can be simplified:

```suggestion
              renderFunction: (record: any) => `${record.namespace}/${record.tenant}`,
```

**Review by bexsoft — CHANGES_REQUESTED** [review]
Beside of that, works & LGTM

File: portal-ui/src/screens/Console/Storage/StoragePVCs.tsx (hunk 5 above)
Comment (adfost) [reply portal-ui/src/screens/Console/Storage/StoragePVCs.tsx]: This is outdated after fix.

**Review by dvaldivia — APPROVED** [review]
LGTM.

**Review by bexsoft — APPROVED** [review]
LGTM

**Pull request merged**